packages/office-ui-fabric-react/src/components/Callout/Callout.types.ts

```diff
@@ -192,4 +194,90 @@ export interface ICalloutProps extends React.Props<Callout | CalloutContent> {
    * Callback when the Callout body is scrolled.
    */
   onScroll?: () => void;
+
+  /**
+   * Optional theme for component
+   */
+  theme?: ITheme;
+
+  /**
+  * Optional styles for the component.
+  */
+  getStyles?: IStyleFunction<ICalloutContentStyleProps, ICalloutContentStyles>;
+}
+
+export interface ICalloutContentStyleProps {
+  /**
+   * Theme to apply to the calloutContent.
+   */
+  theme: ITheme;
+
+  /**
+   * Width for callout including borders.
+   */
+  calloutWidth?: number;
+
+  /**
+ * CSS class to apply to the callout.
+ */
+  className?: string;
+
+  /**
+   * Callout positioning data
+   */
+  positions?: ICalloutPositionedInfo;
+
+  /**
+   * Whether or not to clip content of the callout,
+   * if it overflows vertically.
+   */
+  overflowYHidden?: boolean;
+
+  /**
+   * Max height applied to the content of a callout.
+   */
+  contentMaxHeight?: number;
+
+  /**
+   * Background color for the beak and callout.
+   */
+  backgroundColor?: string;
+
+  /**
+   * Width of Callout beak
+   */
+  beakWidth?: number;
+
+  /**
+    * Deprecated at v0.59.1, to be removed at >= v1.0.0. Pass in a beakWidth to dictate size.
+    * @deprecated
+    */
+  beakStyle?: string;
```

Comment:
This has been deprecated in the props, but is still being used in the getBeakStyle func. Is it safe to remove from props, styleprops and func or are there still concerns of this being breaking change? Comment makes me think that it is okay to remove now

Reply:
Good q for @joschect

Reply:
@joschect  sounds good! I'll leave it in.